java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S6818.html

```diff
@@ -0,0 +1,122 @@
+<h2>Why is this an issue?</h2>
+<p><code>@Autowired</code> is an annotation in the Spring Framework for automatic dependency injection. It tells Spring to automatically provide the
+required dependencies (such as other beans or components) to a class’s fields, methods, or constructors, allowing for easier and more flexible
+management of dependencies in a Spring application. In other words, it’s a way to wire up and inject dependencies into Spring components
+automatically, reducing the need for manual configuration and enhancing modularity and maintainability.</p>
+<p>In any bean class, only one constructor is permitted to declare <code>@Autowired</code> with the <code>required</code> attribute set to true. This
+signifies the constructor to be automatically wired when used as a Spring bean. Consequently, when the required attribute remains at its default value
+(true), only a singular constructor can bear the <code>@Autowired</code> annotation. In cases where multiple constructors have this annotation, they
+must all specify <code>required=false</code> to be eligible as candidates for auto-wiring.</p>
+<h2>How to fix it</h2>
+<p>To maintain code clarity and ensure that the Spring context can create beans correctly, have only one constructor annotated with
+<code>@Autowired</code> within a Spring component.</p>
```

Comment:
To be added to the RSPEC of course
```suggestion
<code>@Autowired</code> within a Spring component or set <code>required = false</code>.</p>
```